time-it-builder/src/lib.rs

```diff
@@ -32,20 +36,26 @@ impl TimeItBuilder {
 
     /// This function is called when the timer is done.
     /// The first argument is the name of the timer, the second is the duration.
-    pub fn time_it(&self, action: fn(Duration) -> ()) -> &Self {
+    pub fn time_it(&self, action: TimeItCallBack) -> &Self {
         let mut config = TIMER_CONFIG.write();
         config.action = action;
         self
     }
+
+    pub fn default_tag(&self, tag: impl ToString) -> &Self {
+        let mut config = TIMER_CONFIG.write();
+        config.default_tag = tag.to_string();
+        self
+    }
 }
 
-pub fn get_config() -> Arc<TimeItConfig> {
-    Arc::clone(LAZY_TIMER_CONFIG.get_or_create(|| Arc::new(TIMER_CONFIG.read().clone())))
+pub fn get_config() -> &'static TimeItConfig {
```

Comment:
What are the implications of changing from `Arc` to static ref?

Reply:
After consulting chatgpt, i believe that this is more appropriate because we just have a static (non changing) struct for the lifetime of the program. Which is consistent with what the config is.

Reply:
in this case the `'static` lifetime just means the reference is valid for the lifetime of the program

Reply:
which is already how long the ref was valid for we just needed to tell the compiler that in the fn sig